### PR TITLE
Support for Russian language 

### PR DESCRIPTION
**Pull request by yasdf007**
### Filo learnt some Russian and wants to meet more comrades!

[Klasa](https://github.com/dirigeants/klasa) was translated by @4erem6a, huge thanks to them for doing that.

Every Filo-specific function was translated by me, and I'll be very grateful if you all, Filo's Russian users, review my work and correct me. 

image

### CODE REVIEW

File: languages/ru-RU.js [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+			RESOLVER_MINMAX_MIN: (name, min, suffix) => `${name} должен быть больше ${min}${suffix === this.language.RESOLVER_STRING_SUFFIX ? `${suffix}ов` : suffix}.`,
+			RESOLVER_MINMAX_MAX: (name, max, suffix) => `${name} должен быть меньше ${max}${suffix === this.language.RESOLVER_STRING_SUFFIX ? `${suffix}ов` : suffix}.`,
+			REACTIONHANDLER_PROMPT: 'На какую страницу вы бы хотели перейти?',
+			COMMANDMESSAGE_MISSING: 'Отсутствует один или больше обязательных аргументов после окончания ввода.',
+			COMMANDMESSAGE_MISSING_REQUIRED: (name) => `${name} является обязательным аргументом.`,
+			COMMANDMESSAGE_MISSING_OPTIONALS: (possibles) => `Отсутствует обязательный параметр: (${possibles})`,
+			COMMANDMESSAGE_NOMATCH: (possibles) => `Ваш вариант не соответствует ни одному из возможных: (${possibles})`,
+			// eslint-disable-next-line max-len
+			MONITOR_COMMAND_HANDLER_REPROMPT: (tag, error, time, abortOptions) => `${tag} | **${error}** | Ответьте на это сообщение допустимым значением аргумента в течение **${time}** ${this.pluralize(time, 'секунды', 'секунд', 'секунд')} или отправьте **${abortOptions.join('**, **')}** для отмены.`,
+			// eslint-disable-next-line max-len
```
Comment (emy) [inline languages/ru-RU.js]: redundant because the maxlen is ignored file wide at the top

File: languages/ru-RU.js [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+			RESOLVER_MINMAX_EXACTLY: (name, min, suffix) => `${name} должен быть равен ${min}${suffix === this.language.RESOLVER_STRING_SUFFIX ? `${suffix}ам` : suffix}.`,
+			RESOLVER_MINMAX_BOTH: (name, min, max, suffix) => `${name} должен быть между ${min} и ${max}${suffix === this.language.RESOLVER_STRING_SUFFIX ? `${suffix}ами` : suffix}.`,
+			RESOLVER_MINMAX_MIN: (name, min, suffix) => `${name} должен быть больше ${min}${suffix === this.language.RESOLVER_STRING_SUFFIX ? `${suffix}ов` : suffix}.`,
+			RESOLVER_MINMAX_MAX: (name, max, suffix) => `${name} должен быть меньше ${max}${suffix === this.language.RESOLVER_STRING_SUFFIX ? `${suffix}ов` : suffix}.`,
+			REACTIONHANDLER_PROMPT: 'На какую страницу вы бы хотели перейти?',
+			COMMANDMESSAGE_MISSING: 'Отсутствует один или больше обязательных аргументов после окончания ввода.',
+			COMMANDMESSAGE_MISSING_REQUIRED: (name) => `${name} является обязательным аргументом.`,
+			COMMANDMESSAGE_MISSING_OPTIONALS: (possibles) => `Отсутствует обязательный параметр: (${possibles})`,
+			COMMANDMESSAGE_NOMATCH: (possibles) => `Ваш вариант не соответствует ни одному из возможных: (${possibles})`,
+			// eslint-disable-next-line max-len
```
Comment (emy) [inline languages/ru-RU.js]: redundant because the maxlen is ignored file wide at the top

**Review by emy — CHANGES_REQUESTED** [review]
address commented issues please. otherwise looks good :)

**Comment by yasdf007** [comment]
removed redundant rules, everything seems to be ok now (in code, translation is still needs to be reviewed by end users).

**Review by emy — APPROVED** [review]
Approved.

**Comment by emy** [comment]
спасибо @4erem6a и @yasdf007 ! 👍

**Pull request merged**